stream-chat-android-sample/src/main/kotlin/io/getstream/chat/sample/feature/startup/StartupFragment.kt

```diff
@@ -19,4 +30,44 @@ class StartupFragment : Fragment() {
             }
         )
     }
+
+    private fun showDeprecationWarningIfNeeded() {
+        val context = requireContext()
+
+        if (warningShown(context)) {
+            return
+        }
+
+        val checkbox = CheckBox(context).apply { text = "Do not show this again" }
+        val frame = FrameLayout(context).apply {
+            setPadding(Utils.dpToPx(16), Utils.dpToPx(12), Utils.dpToPx(16), 0)
+        }
+        frame.addView(checkbox)
```

Comment:
Artisinal, organic, hand-crafted custom layout

Reply:
You forgot gluten free

Reply:
That also happens to be true!

Reply:
Looks like a code from somebody who started to learn Compose 😅